website/www/site/content/en/blog/kafka-to-pubsub-example.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title:  "Example to ingest data from Apache Kafka to Google Cloud Pub/Sub"
+date:   2021-01-15 00:00:01 -0800
+categories:
+  - blog
+  - java
+authors:
+  - arturkhanin
+  - ilyakozyrev
+  - alexkosolapov
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+In this blog post we present an example that creates a pipeline to read data from a single topic or
+multiple topics from [Apache Kafka](https://kafka.apache.org/) and write data into a topic
+in [Google Pub/Sub](https://cloud.google.com/pubsub). The example provides code samples to implement
+simple yet powerful pipelines and also provides an out-of-the-box solution that you can just _"
+plug'n'play"_.
+
+This end-to-end example is included
+in [Apache Beam release 2.27](https://beam.apache.org/blog/beam-2.27.0/)
+and can be downloaded [here](https://beam.apache.org/get-started/downloads/#2270-2020-12-22).
+
+We hope you will find this example useful for setting up data pipelines between Kafka and Pub/Sub.
+
+# Example specs
+
+Supported data formats:
+
+- Serializable plain text formats, such as JSON
+- [PubSubMessage](https://cloud.google.com/pubsub/docs/reference/rest/v1/PubsubMessage).
```

Comment:
Nit: You don't need any end punctuations for these list items. 

More information here: https://developers.google.com/style/lists#numbered,-lettered,-and-bulleted-lists

Reply:
Thank you! Fixed it.